app/services/analytics_service.rb

```diff
@@ -46,30 +36,30 @@ def totals
   def stats_grouped_by_day
     final_hash = {}
 
-    dates.each do |date|
-      string_date = date.strftime("%a, %m/%d")
-      reaction_data_of_date = reaction_data.select { |rxn| rxn.created_at.beginning_of_day.to_i == date.to_i }
-      logged_in_page_view_data = page_view_data.select { |pv| pv.created_at.beginning_of_day.to_i == date.to_i && pv.user_id.present? }
+    (start_date.to_date..end_date.to_date).each do |date|
+      string_date = date.strftime("%a, %m/%d") # this is locale dependent!
```

Comment:
I agree with changing this to ISO `yyyy-mm-dd`

Reply:
Ok let me do that